workflows4s-web-api-server/src/main/scala/workflows4s/web/api/service/RealWorkflowService.scala

```diff
@@ -30,12 +29,19 @@ class RealWorkflowService(
       json  <- getRealInstanceProgressJson(entry, instanceId)
     } yield json
 
-  def getProgressAsMermaid(definitionId: String, instanceId: String): IO[String] =
+  
+  def createTestInstance(definitionId: String): IO[WorkflowInstance] =
```

Comment:
Do you actually need that operation? Wouldnt it be enough to generate an id on the ui and ask for that instance?